controllers/storagecluster/cephcluster.go

```diff
@@ -490,10 +490,13 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, serverVersion *v
 					KernelMountOptions: getCephFSKernelMountOptions(sc),
 				},
 			},
-			WaitTimeoutForHealthyOSDInMinutes: getWaitTimeoutForHealthOSD(sc),
 		},
 	}
 
+	if sc.Spec.ManagedResources.CephCluster.WaitTimeoutForHealthyOSDInMinutes != 0 {
```

Comment:
This will be set in every case on the CephCluster CR whenever this setting is on the StorageCluster CR, right? From [Petr's comment](https://github.com/red-hat-storage/ocs-operator/pull/2579#discussion_r1620875815), he saw it wasn't always applied previously, but perhaps this will fix it as well?

Reply:
Yes, this should set it always whenever a change is applied on the storageCluster CR.
Infact, it should have been previously as well 🤔